python/tvm/relay/frontend/pytorch.py

```diff
@@ -1769,10 +1780,27 @@ def get_v(v, default_v):
                 return v
             return default_v
 
-        amin = get_v(inputs[1], np.finfo(np.float32).min)
-        amax = get_v(inputs[2], np.finfo(np.float32).max)
+        if min_only:
+            amin = get_v(inputs[1], np.finfo(np.float32).min)
```

Comment:
It is not introduced in this PR, but is using float32's max prudent here? We might have all sorts of dtypes as inputs, also `-inf` should probably not be clamped to `-3.4028235e+38`.

Reply:
Good catch, fixed but not sure what to do with clamping `inf.`

Reply:
I think -inf should stay -inf if we clamp with only max. PyTorch uses separate implementations kernels for this (kind of the reverse of what we do here), I don't know if that might be a good choice for TVM.

Reply:
I see, that makes sense. Probably the best way for us is to allow `None` value in `clip` op for one-way clipping, and do only `max` or `min inside topi. I left a TODO item for that.